.github/workflows/release.yml

```
@@ -0,0 +1,62 @@
+        with:
+          node-version: '20'
+          cache: 'pnpm'
+          
+      - name: Install dependencies
+        run: pnpm install
+        
+      - name: Build VSCode Extension Package
+        run: pnpm build:vscode-package
+        
+      - name: Upload VSIX to GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)